packages/cf-component-box/src/propertiesSpec.js

```diff
@@ -0,0 +1,342 @@
+import React, { PropTypes } from 'react';
+
+const numberOrString = PropTypes.oneOfType([
+  PropTypes.string,
+  PropTypes.number
+]);
+
+// TODO: This needs to be pulled from a proper single source of truth
+const baseFontSize = 15;
+const pxToRem = pixels => {
```

Comment:
I would rather use the helper from polished so we can easily compile it away with https://github.com/styled-components/babel-plugin-polished